trinity/tx_pool/pool.py

```diff
@@ -41,9 +42,17 @@ class TxPool(BaseService, PeerPoolSubscriber):
     """
     logger = logging.getLogger("trinity.tx_pool.TxPool")
 
-    def __init__(self, peer_pool: PeerPool, token: CancelToken = None) -> None:
+    def __init__(self,
+                 peer_pool: PeerPool,
+                 tx_validation_fn: Callable[[BaseTransactionFields], bool],
+                 token: CancelToken = None) -> None:
         super().__init__(token)
         self._peer_pool = peer_pool
+
+        if tx_validation_fn is None:
+            raise ValueError('Must pass a validator instance')
```

Comment:
This exception message looks like it needs to be updated.

Reply:
Yep.